preset/variables.scss

```diff
@@ -10,6 +10,7 @@
 $body-font-family: 'proxima-nova', 'helvetica' , sans-serif;
 $heading-font-family: 'freight-sans-pro', 'helvetica', sans-serif;
 $border-radius-root: 5px;
+$btn-letter-spacing: 'unset';
```

Comment:
We've removed the letter spacing below: https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/blob/master/preset/variables.scss#L133

Reply:
Sweet removed here https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/pull/10/commits/6c263fb6b33c5b8f3f67b676188695bebe45f5ed